scenarios/evaluate/Supported_Evaluation_Metrics/Agent_Evaluation/Evaluate_SK_Chat_Completion_Agent.ipynb

```diff
@@ -0,0 +1,475 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "bf5280e2",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "source": [
+    "# Evaluate Semantic Kernel AI (ChatCompletion) Agents in Azure AI Foundry"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0330c099",
+   "metadata": {},
+   "source": [
+    "## Objective\n",
+    "\n",
+    "This sample demonstrates how to evaluate Semantic Kernel AI ChatCompletionAgents in Azure AI Foundry. It provides a step-by-step guide to set up the environment, create an agent, and evaluate its performance."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b364c694",
+   "metadata": {},
+   "source": [
+    "## Time\n",
+    "You can expect to complete this sample in approximately 20 minutes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "919c6017",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "- api_key\n",
+    "chat_deployment_name\n",
+    "endpoint\n",
+    "\n",
+    "## Prerequisites\n",
+    "### Packages\n",
+    "- `semantic-kernel` installed (`pip install semantic-kernel`)\n",
+    "- `azure-ai-evaluation` SDK installed\n",
+    "- An Azure OpenAI resource with a deployment configured\n",
+    "\n",
+    "### Environment Variables\n",
+    "- For AzureChatService:\n",
+    "  - `api_key`\n",
+    "  - `chat_deployment_name`\n",
+    "  - `endpoint`\n",
+    "- For evaluating agents:\n",
+    "  - `AZURE_OPENAI_ENDPOINT`\n",
+    "  - `AZURE_OPENAI_API_KEY`\n",
+    "  - `AZURE_OPENAI_API_VERSION`\n",
+    "  - `MODEL_DEPLOYMENT_NAME`\n",
+    "- For Azure AI Foundry (Bonus):\n",
+    "  - `AZURE_SUBSCRIPTION_ID`\n",
+    "  - `PROJECT_NAME`\n",
+    "  - `RESOURCE_GROUP_NAME`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ba1d6576",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "source": [
+    "### Create a AzureChatCompletion service - [reference](https://learn.microsoft.com/en-us/semantic-kernel/concepts/ai-services/chat-completion/?tabs=csharp-AzureOpenAI%2Cpython-AzureOpenAI%2Cjava-AzureOpenAI&pivots=programming-language-python)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "518475c7",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Load environment variables from .env file\n",
```

Comment:
Lets remove this since this adds python-dotenv as dependency. We expect customer to have set env variables, how they do it , customers make that decision.